images/build/Dockerfiles/cli.Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG DEBIAN_FLAVOR
-FROM debian:${DEBIAN_FLAVOR}-slim
+FROM buildpack-deps:${DEBIAN_FLAVOR}-curl
```

Comment:
what was the impact in terms of size?

Reply:
From roughly 200MB to 275MB -- this change was needed since the current `debian` base we're using for the CLI image wasn't supporting dynamic installation, and we were looking to create a lightweight CLI image that could support this for the `oryx dockerfile` scenario.

Reply:
@arroyc welcome back 😄